tests/resource/test_resource.py

```diff
@@ -220,8 +220,8 @@ class _CustomOpWithoutResource(Operation):  # pylint: disable=too-few-public-met
     Resources(2, 2, {"Hadamard": 1, "CNOT": 1}, 2, 0),
     Resources(3, 3, {"PauliZ": 1, "CNOT": 1, "RX": 1}, 2, 10),
     Resources(2, 6, {"Hadamard": 3, "RX": 2, "CNOT": 1}, 4, 100),
-    Resources(2, 5, {"Hadamard": 1, "CNOT": 1, "Identity": 1, "PauliZ": 2}, None, 0),
-    Resources(3, 7, {"PauliZ": 3, "CNOT": 1, "RX": 1, "Identity": 1, "CustomOp2": 1}, None, 10),
+    Resources(2, 5, {"Hadamard": 1, "CNOT": 1, "Identity": 1, "PauliZ": 2}, 5, 0),
```

Comment:
Might be good to add a comment here defining these numbers, e.g.,
`# wires, gates, gate_types, depth, shots`

Reply:
Added 👍🏼